controllers/cartController.js

```diff
@@ -0,0 +1,63 @@
+const cartService = require('../services/cartService');
+const { catchError } = require('../middlewares/error');
+
+const createCartItem = catchError(async (req, res) => {
+  try {
+    const { productId, quantity } = req.body;
+
+    const userId = req.user.id;
+
+    const cartItem = await cartService.createCartItem({
+      userId,
+      productId,
+      quantity,
+    });
+    res.status(201).json({ message: cartItem });
+  } catch (err) {
+    res.status(400).json({ message: err.message });
+  }
+});
+
+const getCartList = async (req, res) => {
+  try {
+    const userId = req.user.id;
+    const cartList = await cartService.getCartList(userId);
+    res.status(201).json({ cartList });
```

Comment:
```suggestion
    res.status(200).json({ cartList });
```
http request method 중 GET 과 관련된 더 적합한 status code를 사용할 수 있도록 해주세요.